ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/DisplayGrid/DisplayGrid.cpp

```diff
@@ -61,7 +62,6 @@ void DisplayGrid::componentComplete()
   QQuickItem::componentComplete();
 
   // find QML MapView component
-  m_mapView = findChild<MapQuickView*>("mapView");
```

Comment:
I think we need a ptr check now: this way the code will crash if the qml does not set the `mapView` correctly.

I also think all of the subsequent logic should be moved out of component complete and into the `setMapView` call or similar

Reply:
Added this comment to the larger mapview-access ticket.

Reply:
Added this comment to the larger mapview-access ticket.

Reply:
Added this comment to the larger mapview-access ticket.

Reply:
Added this comment to the larger mapview-access ticket.

Reply:
Added this comment to the larger mapview-access ticket.